Makefile

```diff
@@ -74,3 +73,4 @@ down:
 dev-teardown:
 	docker-compose -p athensdev down
 	docker volume prune
+	docker network prune
```

Comment:
this should already happen with `docker-compose -p athensdev down` -- if not, let's do it in another PR :)